DNNCommunity/Dnn.CommunityForums
# ENH: Remove plaintext email templates

### Description of PR...
Community Forums 08.00.00 has substantial improvements for template parsing, caching, and editing (e.g. you can now edit a template directly in the file system rather than using the Forums Control Panel). Community Forums used to support plain-text vs HTML email templates, but never actually used them. All emails are using the HTML template. This enhancement removes the html vs plain-text differences in the email templates, internal code, and converts existing templates during a 08.00.00 upgrade.

## Changes made
- change template editor to use just HTML rather than HTML and plaintext templates
- change template internals to no longer store &lt;html&gt; vs &lt;plaintext&gt; content tags with template
- don't encode/decode HTML templates 
- removed plaintext email templates from default configuration
- deprecated (public) methods with both HTML and plaintext parameters and created new methods 
- updated code to no longer call deprecated methods

## How did you test these updates?  
- Tested a new install and against copy of dnncommunity.org
  - ensure that default templates are properly loaded during a new install
  - ensure that existing templates are properly upgraded from 07.00.12
  - ensure that template editor in control panel functions properly without HTML/text email templates
  - ensure basic forums functionality (post topic/reply, etc.)
  - ensure that email notifications are properly sent out with HTML templates
  
## PR Template Checklist



## Please mark which issue is solved


Close #450
